README.md

```diff
@@ -57,9 +58,7 @@ a. Place the test file test_viewmodel.py in test folder of project app and execu
 poetry run pytest test_viewmodel.py
 
 2) Running Integration tests
-a. Place test file test_trello.py in test folder and run  poetry run pytest test_trello.py
```

Comment:
We shouldn't just remove the reference to the test, let's delete the test file itself! Otherwise a naive `poetry run pytest` will see failures that are likely to confuse